### DISCUSSION TITLE
How to have an external link ? to open in a newt tab ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [requarks/wiki#5227] [Help / Questions] [answered]
Hello , 

What should I do to have a an external link ? in WIKIjs ? 

I really need your help ?

Best Regards

**Comment by NGPixel** [comment, marked as answer]
You need to enable "Open external links in a new tab" in the Administration area > rendering:

<img width="941" alt="image" src="https://user-images.githubusercontent.com/15522395/166085728-7c3cdc7e-7f37-40b2-bd8d-0241a9fe53ab.png">
